tests/ctst/world/Zenko.ts

```diff
@@ -58,38 +137,40 @@ export default class Zenko extends World {
         super(options);
 
         // store service users credentials from world parameters
-        if (this.parameters.ServiceUsersCredentials) {
-             Object.entries(JSON.parse(this.parameters.ServiceUsersCredentials))
-                .forEach((entry : [string, any]) => {
-                    Zenko.serviceUsersCredentials[entry[0] as string] = {
-                        AccessKeyId: entry[1].accessKey as string,
-                        SecretAccessKey: entry[1].secretKey as string,
-
+        if ((this.parameters as Parameters).ServiceUsersCredentials) {
+            Object.entries(JSON.parse(
+                (this.parameters as Parameters).ServiceUsersCredentials as string) as UserCredentials)
+                .forEach(entry => {
+                    Zenko.serviceUsersCredentials[entry[0]] = {
+                        AccessKeyId: (entry[1] as {accessKey: string}).accessKey,
+                        SecretAccessKey: (entry[1] as {secretKey: string}).secretKey,
                     };
                 });
         }
 
+        const worldParameters = this.parameters as Parameters;
         // Workaround to be able to access global parameters in BeforeAll/AfterAll hooks
-        CacheHelper.parameters = this.parameters;
-        this.cliMode.parameters = this.parameters;
-
-        if (this.parameters.AccountSessionToken) {
-            CacheHelper.ARWWI[CacheHelper.AccountName] = {
-                AccessKeyId: this.parameters.AccountAccessKey,
-                SecretAccessKey: this.parameters.AccountSecretKey,
-                SessionToken: this.parameters.AccountSessionToken,
+        CacheHelper.parameters = worldParameters;
+        const cacheParameters = CacheHelper.parameters as Parameters;
+        this.cliMode.parameters = worldParameters as ClientOptions;
+
+        if (worldParameters.AccountSessionToken) {
+            ((CacheHelper.ARWWI as { [key:string]: unknown })[CacheHelper.AccountName]) = {
+                AccessKeyId: worldParameters.AccountAccessKey,
+                SecretAccessKey: worldParameters.AccountSecretKey,
+                SessionToken: worldParameters.AccountSessionToken,
             };
-            CacheHelper.parameters.AccessKey = this.parameters.AccountAccessKey;
-            CacheHelper.parameters.SecretKey = this.parameters.AccountSecretKey;
-            CacheHelper.parameters.SessionToken = this.parameters.AccountSessionToken;
+            cacheParameters.AccessKey = worldParameters.AccountAccessKey;
+            cacheParameters.SecretKey = worldParameters.AccountSecretKey;
+            cacheParameters.SessionToken = worldParameters.AccountSessionToken;
         } else {
-            CacheHelper.AccountName = CacheHelper.parameters.AccountName;
-            CacheHelper.parameters.AccessKey = this.parameters.AccountAccessKey;
-            CacheHelper.parameters.SecretKey = this.parameters.AccountSecretKey;
+            CacheHelper.AccountName = cacheParameters.AccountName as string;
```

Comment:
From lines above
```
CacheHelper.parameters = worldParameters
```

So why not using `worldParameters` direclty? Also, AccountName should already be a string in the type.

Reply:
Responding to here and above: admittedly I have no idea why we make these assignments as the variables are not even used later in the code. I'll drop them and let's see what happens.
b777f3979d70fe34f44b7939b05fd8751d4d61f9